sdks/python/apache_beam/runners/direct/watermark_manager.py

```diff
@@ -144,8 +144,10 @@ def _refresh_watermarks(self, applied_ptransform, side_inputs_container):
             for consumer in consumers:
               unblocked_tasks.extend(
                   self._refresh_watermarks(consumer, side_inputs_container))
+      # notify the side_inputs_container
```

Comment:
<!--new_thread; commit:0d82258b7674656d3a6f86748f47d23de8cf512f; resolved:0-->
Please fix capitalization, period end of sentence.

Reply:
Done.